addons/l10n_ec/models/l10n_ec_sri_payment.py

```diff
@@ -11,3 +11,4 @@ class SriPayment(models.Model):
 
     name = fields.Char("Name", translate=True)
     code = fields.Char("Code")
+    active = fields.Boolean("Active", default=True)
```

Comment:
@jco-odoo I think the field should be available in the form view, so the user could reactivate any record needed, or deactivate the ones he doesn't use in his business case.

Reply:
Maybe it's better to add a default filter option @vbe-odoo @jco-odoo @pepetreshere, like Odoo manages records with an active field, to show active or archived records. In addition, Odoo already creates automatic archiving or unarchive actions, and by creating the filter, it complements itself.

Reply:
Indeed, that actually a good idea, so users will only see the active ones by default with the filter, and if they remove it they will see all of the records.

Reply:
I did the opposite: it shows all and you see if they are active or not.  As we do not have many, it does not seem needed to add a filter by default.